storage/innobase/include/os0file.h

```diff
@@ -551,9 +551,10 @@ os_file_create_simple_no_error_handling_func(
 	ibool*		success)/*!< out: TRUE if succeed, FALSE if error */
 	MY_ATTRIBUTE((nonnull, warn_unused_result));
 /****************************************************************//**
-Tries to disable OS caching on an opened file descriptor. */
+Tries to disable OS caching on an opened file descriptor.
+@return TRUE if operation is success and FALSE otherwise */
 UNIV_INTERN
-void
+ibool
```

Comment:
C++ "bool" in 5.6+, s/TRUE/true, FALSE/false in the header comment above and in the code